forward43/setup_elasticsearch.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+import pandas as pd
+
+import utils.elasticsearch as es_utils
+
+
+def create_index_with_mapping(es):
+
+    # FIXME: The mappings should be retrieved from the mappings PR
+    mappings = {
+        "properties": {
+            "name": {
+                "type": "keyword"
+            },
+            "creator": {
+                "type": "text"
+            },
+            "pledged": {
+                "type": "float"
+            },
+            "state": {
+                "type": "text"
+            }
+        }
+    }
+
+    is_created = es_utils.create_index(
+        es_object  = es, 
+        index_name = 'forward43', 
+        mappings   = mappings,
+        n_shards   = 1,
+        n_replicas = 0
+    )
+
+
+def read_json_data(es):
+
+    # FIXME: This assumes that data is stored in ../data folder. Only kickstarter_scraper does this.
+    #        1. Do the same for the other scrapers
+    #        2. Add a path argument
+    data_path_common = '../data/'
+
+    for _file in os.listdir(data_path_common):
+
+        if _file.endswith('.json'):
+            with open(data_path_common + _file) as f:
```

Comment:
Doublecheck that there will only be one version per crawler in the dir, otherwise you'll get some clashing data here